### PR TITLE
code review

### PR DESCRIPTION
**Pull request by PriyankaKarthikeyan**

### CODE REVIEW

File: ApiGateway/pom.xml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<groupId>com.stackroute.findMeClinic</groupId>
+	<artifactId>ApiGateway</artifactId>
+	<version>0.0.1-SNAPSHOT</version>
+	<packaging>jar</packaging>
+
+	<name>ApiGateway</name>
+	<description>Demo project for Spring Boot</description>
```
Comment (slswamyj) [inline ApiGateway/pom.xml]: Use proper project description

File: ConfigService/pom.xml [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<groupId>com.stackroute.findMeClinic</groupId>
+	<artifactId>ConfigService</artifactId>
+	<version>0.0.1-SNAPSHOT</version>
+	<packaging>jar</packaging>
+
+	<name>ConfigService</name>
+	<description>Demo project for Spring Boot</description>
```
Comment (slswamyj) [inline ConfigService/pom.xml]: use proper project description

File: DoctorAuth/pom.xml [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<groupId>com.stackroute.findMeClinic</groupId>
+	<artifactId>DoctorAuth</artifactId>
+	<version>0.0.1-SNAPSHOT</version>
+	<packaging>jar</packaging>
+
+	<name>DoctorAuth</name>
+	<description>Demo project for Spring Boot</description>
```
Comment (slswamyj) [inline DoctorAuth/pom.xml]: Use proper project description

File: DoctorAuth/src/main/java/com/stackroute/findmeclinic/doctorauth/exception/DoctorAlreadyExistsEcxeption.java [hunk 4]
```diff
@@ -0,0 +1,11 @@
+package com.stackroute.findmeclinic.doctorauth.exception;
+
+public class DoctorAlreadyExistsEcxeption extends Exception {
```
Comment (slswamyj) [inline DoctorAuth/src/main/java/com/stackroute/findmeclinic/doctorauth/exception/DoctorAlreadyExistsEcxeption.java]: Class name spelling is wrong.

File: PatientAuth/bin/src/main/resources/application.yml [hunk 5]
```diff
@@ -0,0 +1,22 @@
+spring:
```
Comment (slswamyj) [inline PatientAuth/bin/src/main/resources/application.yml]: Do we need separate authentication services for Doctor/Hospital or Patients.?

Comment (jasonchristopher61) [reply PatientAuth/bin/src/main/resources/application.yml]: For the initial sprints we need separate services for authentication of doctors and patients.. but in the end we will find a mechanism to separate doctors from patients and henceforth we will use a single service for authentication.

File: DoctorAuth/src/main/java/com/stackroute/findmeclinic/doctorauth/exception/DoctorAlreadyExistsEcxeption.java (hunk 4 above)
Comment (sivaranjanigs) [reply DoctorAuth/src/main/java/com/stackroute/findmeclinic/doctorauth/exception/DoctorAlreadyExistsEcxeption.java]: changes done sir

**Pull request merged**